src/code/z_scene.c

```diff
@@ -166,11 +166,11 @@ s32 Scene_ExecuteCommands(GlobalContext* globalCtx, SceneCmd* sceneCmd) {
         osSyncPrintf("*** Scene_Word = { code=%d, data1=%02x, data2=%04x } ***\n", cmdCode, sceneCmd->base.data1,
                      sceneCmd->base.data2);
 
-        if (cmdCode == 0x14) {
+        if (cmdCode == SCENE_CMD_ID_END) {
             break;
         }
 
-        if (cmdCode <= 0x19) {
+        if (cmdCode < ARRAY_COUNT(gSceneCmdHandlers)) {
```

Comment:
The size of the extern for `gSceneCmdHandlers` should also be updated to have `SCENE_CMD_ID_MAX` if we want it to shift properly. Or alternatively we could just use `SCENE_CMD_ID_MAX` here and leave the size unset in the extern(s)